ykluks.cfg

```diff
@@ -1,6 +1,6 @@
 # If you change this file, you need to run 
 #   update-initramfs -u
-WELCOME_TEXT="Please insert yubikey and press enter or enter a valid passphrase"
+WELCOME_TEXT="Please insert yubikey and press enter (and optionally the Yubikey button) or enter a valid passphrase"
```

Comment:
see above.
Please use the old text. I would rather add a comment in the config file or adapt the ``WELCOME_TEXT`` if possible depending on the enrolley yubikey mode.